readme.md

```diff
@@ -427,6 +427,18 @@ Router.onAppUpdated = (nextUrl) => {
 }
 ```
 
+##### `withRouter` Higher-Order Component
+
+If you want to pass the `router` object into a component as a prop, you can use the `withRouter` HOC:
+
+```jsx
+import { withRouter } from 'next/router'
+
+const SecretLink = ({router}) => (<a href={`${router.pathname}/secret`}>Secrets!</a>)
+
+export default withRouter(SecretLink)
```

Comment:
I changed the API, but currently options is not being used for anything. How should I use it in the withRouter HOC?